index.html

```diff
@@ -6,6 +6,8 @@
   <meta name="description" content="Deploy a Production Ready Kubernetes Cluster">
   <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <link rel="stylesheet" href="//unpkg.com/docsify-themeable/dist/css/theme-simple.css">
+  <link rel="icon" href="/logo/logo-clear.png" type="image/png" />
+
```

Comment:
Is there a particular reason for that extra space ?

Reply:
No, should I remove it?